src/Components/settings-dropdown/setting-index.tsx

```diff
@@ -0,0 +1,115 @@
+const ORG_URL = "http://localhost:8080/organizations";
+const USR_URL = "http://localhost:8080/users";
```

Comment:
*Actionable*
Please update these variables to use a dynamic base URL. Its not a big deal here because we're using our local but after deploying this to staging, we would have to go in and change these values manually.

Please leverage environment variables for the URI (`http://localhost:8080`). You can still have `const ORG_URL = <env-var-here> + /organizations`. The goal is to make the base URL dynamic based on the environment.